### ISSUE TITLE
test_long_stdlib failure on .NET Core 3.1 + Linux

### ISSUE THREAD

**Issue by slozier** [IronLanguages/ironpython3#980]
Noticed the following failure in `test_long_stdlib` during a CI run on .NET Core 3.1 + Linux:
```
======================================================================
FAIL: test_bitop_identities (test.test_long.LongTest) (n=32, p2=4294967296, x=-79228162514264337593543917568)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vsts/work/1/s/Src/StdLib/Lib/test/test_long.py", line 254, in check_bitop_identities_1
    eq(x // p2, x >> n)
AssertionError: -18446744073709551616 != 0

----------------------------------------------------------------------
Ran 21 tests in 4.823s

FAILED (failures=1, expected failures=4)
  X netcoreapp3.1.IronPython.test_long_stdlib [4s 906ms]
  Error Message:
   Traceback (most recent call last):
  File "/home/vsts/work/1/s/Tests/test_long_stdlib.py", line 45, in <module>
  File "/home/vsts/work/1/s/bin/Release/netcoreapp3.1/Lib/iptest/ipunittest.py", line 183, in run_test
  File "/home/vsts/work/1/s/Src/StdLib/Lib/test/support/__init__.py", line 1787, in _run_suite
  File "/home/vsts/work/1/s/Src/StdLib/Lib/test/support/__init__.py", line 1812, in run_unittest
test.support.TestFailed: Traceback (most recent call last):
  File "/home/vsts/work/1/s/Src/StdLib/Lib/test/test_long.py", line 254, in check_bitop_identities_1
    eq(x // p2, x >> n)
AssertionError: -18446744073709551616 != 0
```

**Comment by slozier** [comment]
Appears to be intermittent, also got it to fail locally (.NET Core 3.1 + Linux) after a few tries:
```
FAIL: test_bitop_identities (test.test_long.LongTest) (n=32, p2=4294967296, x=-18446744073709551615)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/slozier/ironpython3/Src/StdLib/Lib/test/test_long.py", line 254, in check_bitop_identities_1
    eq(x // p2, x >> n)
AssertionError: -4294967296 != 0
```

**Comment by slozier** [comment]
Also occurred on Mono + Linux during CI:
```
FAIL: test_bitop_identities (test.test_long.LongTest) (n=32, p2=4294967296, x=-18446744073709543424)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/ironpython3/ironpython3/Src/StdLib/Lib/test/test_long.py", line 254, in check_bitop_identities_1
    eq(x // p2, x >> n)
AssertionError: -4294967296 != 0
```

**Comment by slozier** [comment]
Also failed on macOS... n=32, p2=4294967296, x=-18446744073709551615

**Comment by BCSharp** [comment]
Isn't solved by #984?

**Comment by slozier** [comment]
Indeed, forgot to tag the PR.
